convnets/logistic_sgd.py

```diff
@@ -428,8 +428,8 @@ def sgd_optimization_mnist(learning_rate=0.13, n_epochs=1000,
         )
         % (best_validation_loss * 100., test_score * 100.)
     )
-    print 'The code run for %d epochs, with %f epochs/sec' % (
-        epoch, 1. * epoch / (end_time - start_time))
+    print('The code run for %d epochs, with %f epochs/sec' % (
+        epoch, 1. * epoch / (end_time - start_time)))
 
     print >> sys.stderr, ('The code ran for %.1fs' % ((end_time - start_time)))
```

Comment:
That's very strange, given that it is even an example of syntax not
supported in Python 3 anymore:
https://docs.python.org/3.0/whatsnew/3.0.html#print-is-a-function :/
On Aug 6, 2015 9:12 AM, "vdumoulin" <notifications@github.com> wrote:

> In convnets/logistic_sgd.py
> <https://github.com/mila-udem/summerschool2015/pull/24#discussion_r36411630>
> :
>
> > @@ -428,8 +428,8 @@ def sgd_optimization_mnist(learning_rate=0.13, n_epochs=1000,
> >          )
> >          % (best_validation_loss * 100., test_score * 100.)
> >      )
> > -    print 'The code run for %d epochs, with %f epochs/sec' % (
> > -        epoch, 1. * epoch / (end_time - start_time))
> > +    print('The code run for %d epochs, with %f epochs/sec' % (
> > +        epoch, 1. * epoch / (end_time - start_time)))
> >
> >      print >> sys.stderr, ('The code ran for %.1fs' % ((end_time - start_time)))
>
> Surprisingly, I had no problem running this line with Python 3.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/summerschool2015/pull/24/files#r36411630>.
>